frontend/stories/components/Header.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { Meta } from "@storybook/react";
+
+import Header from "src/components/Header";
+
+const meta: Meta<typeof Header> = {
+  title: "Components/Layout/Header",
+  component: Header,
+};
+export default meta;
+
+export const Default = {
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov",
```

Comment:
```suggestion
      url: "https://www.figma.com/file/lpKPdyTyLJB5JArxhGjJnE/beta.grants.gov?type=design&node-id=1-1882&mode=design&t=xjsnXcLzBSmyUoXT-4",
```
The more specific figma URL to for the header component will zoom the design panel in to the header, instead of zooming all the way out to see the whole figma.

before:
![image](https://github.com/HHS/grants-equity/assets/25465464/35d039a1-00fa-4e2e-81b6-7e4db187189b)


after:
![image](https://github.com/HHS/grants-equity/assets/25465464/4bfe03cf-27b8-41ec-babf-2ef75845aec1)